packages/bot-web-ui/src/components/trade-animation/trade-animation.scss

```diff
@@ -221,6 +247,8 @@
 
     .notification-close {
         cursor: pointer;
+        transform: translateZ(0);
+        -webkit-filter: invert(1);
         filter: invert(1);
```

Comment:
The problem was related to `filter: invert(1);`
We can see the warning in the safari.
Since the **prefix** didn't help, but I added it anyway, in some cases forcing GPU acceleration can help.
`transform: translateZ(0)` is a trick to fix graphical artifacts, the browser can optimize the rendering of this element using the GPU. CSS filters can be passed from CPU (central processing unit) to GPU (graphics processing unit)